share/spice/indego_bikeshare_locations/indego_bikeshare_locations.js

```diff
@@ -0,0 +1,61 @@
+(function (env) {
+    "use strict";
+
+    env.ddg_spice_indego_bikeshare_locations = function(api_result) {
+        if (!api_result || !api_result.features) {
+            return Spice.failed('indegobikesharelocations');
+        }
+
+        function normalize(item) {
+            if (item.properties.kioskPublicStatus != 'Active') {
```

Comment:
Check `item.properties` first before accessing `item.properties.kioskPublicStatus`. Also `!==` just to be consistent with the rest of the codebase. :)